styles/css/core/variables.css

```diff
@@ -32,7 +32,8 @@
   --pgn-elevation-modal-zindex: 1050;
   --pgn-elevation-modal-backdrop-zindex: 1040;
   --pgn-elevation-dropdown-zindex: 1000;
-  --pgn-transition-collapse: height .35s ease;
+  --pgn-transition-collapse-width: width .35s ease;
```

Comment:
Is there reason behind adding `collapse-width`. Don't remember if we use some width collapse.

Reply:
We have a variable for this case, it looks like you added it :)